OracleWebLogic/samples/12213-domain-home-in-image-ssl/container-scripts/startManagedServer.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Copyright (c) 2014-2018 Oracle and/or its affiliates. All rights reserved.
+#
```

Comment:
This copyright header is missing the the license line: `Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.`